google-cloud-testing/storage-benchwrapper/README.md

```diff
@@ -0,0 +1,67 @@
+# storage-benchwrapper
+
+storage-benchwrapper is a gRPC wrapper around the storage library for benchmarking purposes.
+
+## Running
+
+```
+mvn clean install -DskipTests=true (one time)
+cd google-cloud-testing/storage-benchwrapper
+export STORAGE_EMULATOR_HOST=localhost:8080
+mvn clean install exec:java -DskipTests=true -Dport=8081
+```
+
+## Generating .proto sources
+
+Sources are generated automatically as part of the protobuf-maven-plugin plugin,
+but if you'd like to generate them yourself to see the output you can run:
+
+```
+cd google-cloud-testing/storage-benchwrapper
+protoc \
+    --plugin=protoc-gen-grpc-java=build/exe/java_plugin/protoc-gen-grpc-java \
+    --java_out=src/main/java \
+    --grpc-java_out=src/main/java \
+    --proto_path=src/main/proto \
+    src/main/proto/*.proto
+```
+
+Note that you'll need to delete these, or uncomment the plugin, since multiple
+definitions of the same class is not allowed.
+
+Note that these should not be committed into git.
```

Comment:
You might be able to delete some or all of these "Note that"s.

Reply:
Ack